### ISSUE TITLE
"Subroutine JSON::PP::Boolean::(++ redefined" with newer perls

### ISSUE THREAD

**Issue by ntyni** [rurban/Cpanel-JSON-XS#200]
Hi,

we're seeing this with Perl 5.36 and newer:

```
$ /opt/perl/bin/perl5.37.3 -w -e 'use Cpanel::JSON::XS (); use JSON::PP'
Subroutine JSON::PP::Boolean::(++ redefined at /opt/perl/lib/5.37.3/overload.pm line 52.
Subroutine JSON::PP::Boolean::(0+ redefined at /opt/perl/lib/5.37.3/overload.pm line 52.
Subroutine JSON::PP::Boolean::(-- redefined at /opt/perl/lib/5.37.3/overload.pm line 52.
```

This is with Cpanel::JSON::XS 4.31 and JSON::PP 4.10 or 4.11. Apologies if Cpanel::JSON::XS is not the one to blame :)

The warnings are breaking at least the Request Tracker test suite, as discussed in https://bugs.debian.org/1016806 .

**Comment by rurban** [comment]
Apologies, I didn't test with newer perl's

**Comment by rurban** [comment]
Finally fixed with 4.32
